src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,36 @@
+.contacts {
+    &__content {
+        @include onTablet {
+            @include useGrid (12, 24px);
+        }
+    }
```

Comment:
you only have onTablet - with 12 columns 🤔 
i think it should have both
- onTablet (6 columns)
- onDesktop (12 columns)